engine/common/oncall_gateway/tasks.py

```diff
@@ -12,16 +12,16 @@
 @shared_dedicated_queue_retry_task(
     autoretry_for=(Exception,),
     retry_backoff=True,
-    max_retries=None,
+    max_retries=1000,
```

Comment:
are you sure we need that amount of retries? It will try to retry for weeks before stop

Reply:
Lowered to 100